### ISSUE TITLE
Wrapping does not work when string contains newline

### ISSUE THREAD

**Issue by staeff** [collective/i18ndude#13]
Hello,

I made the observation, that wrapping does not work correctly, when the message string contains an explicit newline character ('\n').
Interestingly I was not able to replicated this false behavior by writing a new test (https://github.com/staeff/i18ndude/commit/8358f1240a3ac3db0d2b0230b8f8f903c95b9a4e). So the problem seems not to occur in `wrap_string` in `utils.py` (https://github.com/collective/i18ndude/blob/master/src/i18ndude/utils.py#L99) but somewhere else along the way.

To replicate the false behavior you could run the following steps. You should see the problem by having a look at the output of the git diff.

```bash
$ git clone https://gist.github.com/b9adc20c725521516318.git i18ndudetest
$ cd i18ndudetest
$ i18ndude sync --wrap --pot test.pot test.po  # i18ndude needs to be in $PATH
$ git diff
 msgid "ok"
 msgstr ""
-"Falls aktiv wird der entsprechende Linkcycle einen Button haben, mit dem man d
+"Falls aktiv wird der entsprechende Linkcycle einen Button haben, mit dem man "
+"durch ihn blättern kann. mit dem man durch ihn blättern kann. mit dem man "
+"durch ihn blättern kann. mit dem man durch ihn blättern kann."

 msgid "problematic"
 msgstr ""
-"Falls aktiv wird der entsprechende \nLinkcycle einen Button haben, mit dem man
+"Falls aktiv wird der entsprechende \n"
+"Linkcycle einen Button haben, mit dem man durch ihn blättern kann. mit dem man
```
I would be willing to investigate this problem further, hints where to look for the error are appreciated.

**Comment by mauritsvanrees** [comment]
Thanks for reporting. I have released 3.4.1 with a fix.

**Comment by staeff** [comment]
@mauritsvanrees - Thanks for the fix and the release. Will there be a release that includes the fix https://github.com/collective/i18ndude/commit/bff29c705a2e632141a1e54fd5cdee8e8867b484 soon?

**Comment by mauritsvanrees** [comment]
Yes. I have now released 3.4.2 with that fix.
